openlibrary/core/lending.py

```diff
@@ -263,28 +263,6 @@ def get_availability_of_editions(ol_edition_ids):
     """
     return get_availability('openlibrary_edition', ol_edition_ids)
 
-@public
-def get_realtime_availability_of_ocaid(ocaid):
```

Comment:
let's make sure we got all the references (one we found was in borrow.py in a critical path to borrowing)

Reply:
Done